libcontainer/configs/cgroup_linux.go

```diff
@@ -69,6 +69,9 @@ type Resources struct {
 	// CPU hardcap limit (in usecs). Allowed cpu time in a given period.
 	CpuQuota int64 `json:"cpu_quota"`
 
+	// CPU hardcap burst limit (in usecs). Allowed accumulated cpu time additionally for burst in a given period.
+	CpuBurst uint64 `json:"cpu_burst"`
```

Comment:
```suggestion
	CpuBurst uint64 `json:"cpu_burst"` //nolint:revive
```

Reply:
emmm, it's weird; it seems not need to add an extra flag to other filed. = =

Reply:
This is because lint-extra is a job which runs more strict linters on new code only. But yes, I agree that if other fields are named `Cpu*`, we should use `Cpu`, too.